src/features/events_list/readme.md

```diff
@@ -0,0 +1,101 @@
+It can also contains a show timeline button, if the episodes timeline feature is enabled.
+
+### CurrentEvent
+
+The `CurrentEvent` component displays details for the currently selected unlisted event. Unlisted event it is an event that is not in the list of events, nut it comes from the URL.
+
+### FeedSelector
+
+The `FeedSelector` component allows users to filter the event list by feed. It displays a list of available feeds and allows the user to select one or more to filter the event list. Will be displayed if feedSelector feature is enabled.
+
+### BBoxFilter
+
+The `BBoxFilter` component allows users to filter the event list by current map view. It displays a button that, when clicked, switches the filter on or off.
+
+### EpisodeTimelineToggle
+
+The `EpisodeTimelineToggle` component is a button that toggles the visibility of the episode timeline for the currently selected event.
+
+### EventListSettingsRow
+
+The `EventListSettingsRow` component is a container for the `FeedSelector` and `BBoxFilter` components. It is used to group these components together in the user interface.
+
+### FullState
+
+The `FullState` component is the main component that displays the event list. It includes sub-components such as `CurrentEvent`, `FeedSelector`, and `BBoxFilter`. It also includes a `Virtuoso` component to efficiently render large lists of events.
+
+### ShortState
+
+The `ShortState` component is an abbreviated version of the event list that is displayed when there is no event selected. It includes a call-to-action button that prompts the user to select an event.
+
+### Analytics
+
+The `Analytics` component displays statistics for an event, such as the affected population and the settled area. It is displayed on the `EventCard` component.
+
+## Props
+
+### EventList
+
+- `onCurrentChange`: A function that is called when the currently selected event changes. The new event ID is passed as an argument.
+- `initialEventId`: The ID of the event to select when the component first loads.
+
+### FullState
+
+- `currentEventId`: The ID of the currently selected event.
+- `onCurrentChange`: A function that is called when the currently selected event changes.
+
+### ShortState
+
+- `hasTimeline`: A boolean value indicating whether the episode timeline feature is enabled.
+- `openFullState`: A function that is called when the user clicks the call-to-action button.
+- `currentEventId`: The ID of the currently selected event.
+
+# Atoms
+
+## `autoSelectEvent`
+
+The `autoSelectEvent` atom is responsible for automatically selecting an event when the `events_list` feature is loaded.
+
+It accomplishes this by subscribing to the `eventListResourceAtom` and `currentEventResourceAtom` atoms. When `eventListResourceAtom` emits a new value, `autoSelectEvent` checks if the `currentEventResourceAtom` has a null value. If `currentEventResourceAtom` is null, `autoSelectEvent` selects the first event in the `eventListResourceAtom`.
```

Comment:
Important detail - this happens only in case if scheduledAutoSelect was switched to true state before.
This needed because we have cases when event not selected and this is ok (sse when scheduledAutoSelect switched)